docs/sdk/v3/guides/liquidity/03-collecting-fees.md

```diff
@@ -0,0 +1,70 @@
+
+We then pass the Position instance as input to the `NonfungiblePositionManager`'s `addCallParameters` function:
+
+```js reference title="Getting the transaction calldata and parameters" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/560fc2e23a4db6a520a89875ab5cdc8606017952/v3-sdk/collecting-fees/src/example/Example.tsx#L146-L153
+```
+
+Note how the function also requires an options object as its second parameter off type [`AddLiquidityOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L77). This is either of type [`MintOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L74) for minting a new position or [`IncreaseOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L75) for adding liquidity to an existing position. In the minting case, we want to populate `MintOptions`.
+
+The function returns the calldata as well as the value required to execute the transaction, which we then execute. The effect of the transaction is to mint a new Position NFT, which should then be visible on the list of position ids.
+
+### Collecting accrued fees from our position by using the `NonfungiblePositionManager`'s `collectCallParameters` to get the data for creating the collect fees transaction, and then executing the transaction
+
+All of the fee collecting logic can be found in the [`collectFees`](https://github.com/Uniswap/examples/blob/560fc2e23a4db6a520a89875ab5cdc8606017952/v3-sdk/collecting-fees/src/example/Example.tsx#L169) function. The function is very similar to `mintPosition`, except for some details that we will go over. Notice how the **Collect Fees** button is disabled until a position is minted.
+
+To start, we construct a call to the `NonfungiblePositionManager`'s `collectCallParameters`'s function that expects an options object of type  [`CollectOptions`](https://github.com/Uniswap/v3-sdk/blob/08a7c050cba00377843497030f502c05982b1c43/src/nonfungiblePositionManager.ts#L105):
+
+```js reference title="Getting the calldata and value for the transaction" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/560fc2e23a4db6a520a89875ab5cdc8606017952/v3-sdk/collecting-fees/src/example/Example.tsx#L177-L188
+```
```

Comment:
The amount conversion here distracts from the example - maybe good to use local variables in the code to better illustrate?